pyperf/_bench.py

```diff
@@ -424,6 +424,39 @@ def median_abs_dev(self):
             raise ValueError("MAD must be >= 0")
         return value
 
+    def required_nsamples(self):
```

Comment:
If you want to add a public function, please document it at: https://pyperf.readthedocs.io/en/latest/api.html#benchmark-class

Reply:
Good point.  I think we do want it to public (for the same reason the other statistics methods are public).